streams-tests/shared/src/test/scala/zio/stream/experimental/ZStreamUtils.scala

```diff
@@ -0,0 +1,13 @@
+package zio.stream.experimental
+
+import zio._
+
+trait ZStreamUtils {
```

Comment:
Why both trait and object?

Reply:
It was that way in the old utils. Should I change it to just object?

Reply:
Yeah, no need for that trait I think. Probably a leftover of the old inheritance based encoding for ZSpecs.

Reply:
Removed the trait.